notebooks/Data_visualization/Data_visualizations.ipynb

```diff
@@ -0,0 +1,474 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    plt.title('Trip Duration - College')
</pre><p>Can we add a more descriptive title here? like Frequency of trip duration for students ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflows/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>